api/goldens/aptos_api__tests__view_function__test_view_aggregator.json

```diff
@@ -0,0 +1 @@
+["10"]
```

Comment:
newline

Reply:
golden tests compare the full file so new line breaks them